README.md

```diff
@@ -524,8 +582,12 @@ let bar = PlistFiles.Stuff.key1
 
 ## Strings
 
-```sh
-swiftgen strings -t structured-swift4 /path/to/Localizable.strings
+```yaml
+fonts:
+  inputs: /path/to/Localizable.strings
+  outputs:
+    templateName: structured-swift4
+    output: Strings.swift
 ```
 
 This will generate a Swift `enum L10n` that will map all your `Localizable.strings` (or other tables) keys to an `enum case`. Additionaly, if it detects placeholders like `%@`,`%d`,`%f`, it will add associated values to that `case`. Note that all dots within the key are converted to dots in code.
```

Comment:
> map your Localizable keys to an enum case

This isn't true anymore 😉 

> To a `static let` constant. And if it detect placeholders […], it will generate a `static func` with the proper sent argument types instead.